install/experimental.yaml

```diff
@@ -0,0 +1,4560 @@
+# Copyright 2022 The Kubernetes Authors.
```

Comment:
These files should probably be included in https://github.com/kubernetes-sigs/gateway-api/blob/master/.gitattributes

Reply:
Actually, I'm starting to think these files shouldn't be checked in at all. I was looking at what other projects do, and really like cert-manager's solution. They attach manifests to the GitHub release to end up with an installation command like `kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml`. They have a lot of tooling to automate this process, but even if we don't get all the automation in place, it seems like it could be a good approach for us.

Reply:
+1 nice find. There sure is a lot of tooling behind it though!

In the end though, the key part is attaching the yaml to the draft github release. See https://github.com/cert-manager/release/blob/cccdd578bf20a63e366563ee3bb18e9c71fdf7a1/cmd/cmrel/cmd/gcb_publish.go#L455-L459

The github CLI does support this in https://cli.github.com/manual/gh_release_create or https://cli.github.com/manual/gh_release_upload